buildscripts/BuildScripts.csproj

```diff
@@ -1,10 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What is this file for, I see dotnet restore being run on it first. Maybe the file should have a more descriptive name.